src/test/java/com/epam/tc/hw2/ExerciseTwoTest.java

```diff
@@ -0,0 +1,87 @@
+        loginDropdown.click();
+        WebElement loginInput = driver.findElement(By.xpath(XPATH_LOGIN_INPUT));
+        loginInput.sendKeys(loginText);
+        WebElement passwordInput = driver.findElement(By.xpath(XPATH_PASSWORD_INPUT));
+        passwordInput.sendKeys(passwordText);
+        WebElement loginSubmit = driver.findElement(By.xpath(XPATH_LOGIN_SUBMIT));
+        loginSubmit.click();
+        WebElement logoutButton = driver.findElement(By.xpath(XPATH_LOGOUT_SUBMIT));
+        boolean logoutButtonIsDisplayed = logoutButton.isDisplayed();
+        Assertions.assertThat(logoutButtonIsDisplayed)
+                .isTrue();
+        //4. Assert Username is loggined
+        WebElement loginName = driver.findElement(By.id(ID_LOGIN_NAME));
+        Assertions.assertThat(loginName.getText())
+                .isEqualTo(LOGIN_ROMAN_IOVLEV);
+        //5. Open through the header menu Service -> Different Elements Page
+        WebElement dropdownServiceMenu = driver.findElement(By.className(ID_DROPDOWN_SERVICE_MENU));
+        dropdownServiceMenu.click();
+        WebElement menuItemDifferentElements = driver.findElement(By.xpath(XPATH_MENU_ITEM_DIFFERENT_ELEMENT));
+        menuItemDifferentElements.click();
+        Assertions.assertThat(driver.getTitle())
+                .isEqualTo("Different Elements");
+        //6. Select checkboxes
+        WebElement checkBoxWater = driver.findElement(By.xpath(XPATH_CHECKBOX_WATER));
+        checkBoxWater.click();
+        WebElement checkBoxWind = driver.findElement(By.xpath(XPATH_CHECKBOX_WIND));
+        checkBoxWind.click();
+        boolean statusCheckBoxWater = checkBoxWater.isSelected();
+        boolean statusCheckBoxWind = checkBoxWind.isSelected();
+        Assertions.assertThat(statusCheckBoxWater)
+                .isTrue();
+        Assertions.assertThat(statusCheckBoxWind)
+                .isTrue();
+        //7. Select radio
+        WebElement radioButtonSilver = driver.findElement(By.xpath(XPATH_RADIOBUTTON_SILVER));
+        radioButtonSilver.click();
+        boolean statusRadioButtons = radioButtonSilver.isSelected();
+        Assertions.assertThat(statusRadioButtons)
+                .isTrue();
+        //8. Select in dropdown
+        WebElement dropdown = driver.findElement(By.xpath(XPATH_DROPDOWN_BLUE));
+        dropdown.click();
+        //9. Assert that
+        //• for each checkbox there is an individual log row and value is corresponded to the status of checkbox
+        //• for radio button there is a log row and value is corresponded to the status of radio button
+        //• for dropdown there is a log row and value is corresponded to the selected value.
+        WebElement logs = driver.findElement(By.xpath(XPATH_LOGS));
+        Assertions.assertThat(logs.getText())
+                .contains("Water")
+                .contains("Wind")
+                .contains("Silver")
+                .contains("Blue");
```

Comment:
It's a really simple assert. 

Log rows are displayed and 
• checkbox name and its status are corresponding to selected
• radio button name and it status is corresponding to selected
• dropdown name and selected value is corresponding to selected

There are no any status checks. No order checks. You can use another locator to find all log webelements and then get text from each of them